apisix/admin/stream_routes.lua

```diff
@@ -78,6 +102,14 @@ local function check_conf(id, conf, need_id)
     return need_id and id or true
 end
 
+function _M.stream_routes()
```

Comment:
What about moving the init (stream_init_worker) to https://github.com/apache/apisix/blob/2f4a4dba2c0cbc426ae99133c2546112ea71dba8/apisix/admin/init.lua#L353?

We should only initialize it when stream_proxy is enabled: https://github.com/apache/apisix/blob/2f4a4dba2c0cbc426ae99133c2546112ea71dba8/apisix/admin/init.lua#L145

Reply:
Modified，please have a look